starter-code/index.js

```diff
@@ -1,10 +1,29 @@
-var SortedList = function() {};
+var SortedList = function() {
+  this.length = 0;
+  this.list = [];
+};
+
+SortedList.prototype.add     = function(item) {
+    this.list.push(item);
+    this.length++;
+    this.list.sort(function(a,b){
+        return a -b;
+    });
+    }
+}
+SortedList.prototype.get     = function(pos) {
+    if(pos < this.list.length){
+        return this.list[pos];
```

Comment:
You want to return the pos - 1 position. E.g.: first element (index 0) if the user input position 1.